tests/test_env_envp.c

```diff
@@ -31,15 +31,15 @@ void	test_env_envp_two(void)
 		printf("This is %lu. symbol: %s\n", i + 1, envp[i]);
 		i++;
 	}
-	ft_envp_destroy(envp);
+	ft_envp_destroy(&envp);
 
 }
 
 void	test_env_envp_empty(void)
 {
 	ft_envp_create(symtab, &envp);
 	printf("This is envp[0]: %p\n", envp[0]);
-	ft_envp_destroy(envp);
+	ft_envp_destroy(&envp);
 }
 
 void	test_env_envp(void)
```

Comment:
Que es eso?

Reply:
I recompiled the test file with tclean and found out that i didn't update an old test: ft_envp_destroy() is from another module. I just removed the ampersand to not pass a ***